modules/openvpn_status/openvpn_status.go

```diff
@@ -0,0 +1,96 @@
+package openvpn_status
+
+import (
+	"os"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+	"github.com/netdata/go.d.plugin/pkg/matcher"
+)
+
+const (
+	defaultFilePath = "/var/log/openvpn/status.log"
+)
+
+func init() {
+	creator := module.Creator{
+		Defaults: module.Defaults{
+			Disabled: true,
+		},
+		Create: func() module.Module { return New() },
+	}
+
+	module.Register("openvpn_status", creator)
```

Comment:
let's rename it to `openvpn_status_log`:
 - (i believe) the default status filename is 'openvpn-status.log'.
 - having in mind that we have `openvpn` collector, adding the `_log` prefix will make it less confusing what is what.